wallets/shared/src/rango.ts

```diff
@@ -235,6 +235,58 @@ export interface WalletConfig {
   isAsyncSwitchNetwork?: boolean;
 }
 
+export type DerivationPath = {
+  id: string;
+  label: string;
+  getValue: (index: string) => string;
+};
+
+export const EVMDerivationPaths: DerivationPath[] = [
```

Comment:
```suggestion
export const EvmDerivationPaths: DerivationPath[] = [
```

Reply:
Fixed

Reply:
Sorry I made a mistake here. it should be evmDerivationPaths (and also `solanaDerivationPaths`) since for values we use UPPERCASE or camelCase only.

Reply:
fixed